.travis/test.sh

```diff
@@ -18,6 +18,8 @@ then
   UT_FLAGS+=" -K combined_modes"
 fi
 
+UT_FLAGS+=" -K mock_read_routes6_windows "
```

Comment:
I think that its is better to keep `regression.uts` free from architecture dependent tests (it is not actually the case). It would be better to move your test into `mock_windows.uts`. We need to do something similar with the `read_routes6` ones.

What do you think ?